### ISSUE TITLE
Missing ICD9/ICD10 hierarchical concepts and associated relationships

### ISSUE THREAD

**Issue by MaximMoinat** [OHDSI/OMOP-Standardized-Vocabularies#40]
Original issue: https://github.com/OHDSI/CommonDataModel/issues/271

Although ICD is not a standard concept, addition of the hierarchy would be useful for classification of conditions. However, as mentioned in the referenced issue, the higher level ICD chapters (I to XXII) and sections (e.g. 'A00-A09') are missing in the OMOP vocabulary. All chapters for ICD10 can be found in [this ICD browser from the WHO](https://icd.who.int/browse10/2016/en).

Would this be a welcome addition to the vocabularies?

Additional consideration; make ICD10 'official' classification concepts for conditions. Currently these are MedDRA terms, but it is not available at every data site because it requires a licence.

**Comment by cgreich** [comment]
Hm, @MaximMoinat. Slippery slope here. 

For the Standardized Vocabularies to work we need to agree on ONE standard concept representing the meaning. When we add ICD10 we kill that and go back to the Babylonian language confusion. Don't forget: Every country has their own ICD10, which makes matters worse. Finally, the ICD10 Hierarchy sucks. There is no proper Description Logic (rules of inheritance of explicitly declared attributes for each concept), and each concept can only have only one parent. The hierarchical choices also often clash with how SNOMED sees the world.

In other words: Don't go there if you want to stay out of major trouble.

**Comment by cgreich** [comment]
Hm, @MaximMoinat. Slippery slope here. 

For the Standardized Vocabularies to work we need to agree on ONE standard concept representing the meaning. When we add ICD10 we kill that and go back to the Babylonian language confusion. Don't forget: Every country has their own ICD10, which makes matters worse. Finally, the ICD10 Hierarchy sucks. There is no proper Description Logic (rules of inheritance of explicitly declared attributes for each concept), and each concept can only have only one parent. The hierarchical choices also often clash with how SNOMED sees the world.

In other words: Don't go there if you want to stay out of major trouble.

**Comment by cukarthik** [comment]
@cgreich, I agree we want to SNOMED to be standard, but I'm thinking it might be useful to have the fully ICD hierarchy, especially if part of it is in there already. I would think we want people to navigate terminologies in the way they are familiar and then direct them to the standard codes. I know the hierarchy can be misleading b/c some standard codes could be missed, but the tools should help us identify those issues.

**Comment by cgreich** [comment]
@cukarthik :
I know. We want that. I want it more than anybody. And we have the hierarchical relationships, look yourself into Athena. But not in the CONCEPT_ANCESTOR table. That only contains standard concepts.

**Comment by jscherdin** [comment]
Going back to my original request. While the relationship data does exist for the ICD9/10 concepts that are in OMOP, the top levels are missing as concepts in OMOP preventing people from showing the entire hierarchy. It seems that adding less than 100 concepts and their relationships into OMOP would be pretty simple and yet provide a huge value to many groups

**Comment by cgreich** [comment]
What's missing, @jscherdin?

**Comment by jscherdin** [comment]
I already created an issue to document this:
https://github.com/OHDSI/CommonDataModel/issues/271

**Comment by cgreich** [comment]
Ah, yes. Will put in.

**Comment by MaximMoinat** [comment]
Any progress on this issue? Liked the discussion, but not sure whether we have agreed that we want this or not. And is any help needed?
@cgreich

**Comment by jscherdin** [comment]
Following up. @cgreich what is the status on this? Is there a timeline?

**Comment by cgreich** [comment]
Let's do next release. Couple weeks or so.
